added support for ubuntu 24.04 and 22.04 with updated docker v27.2

yshashix opened the pull request:
updated following packages:
   - docker v27.2
    - miniconda for python 3.10
    - sqlite3-pcre2 compiled pcre2.so file
#589

wagmarcel reviewed (commented):
Why is the K8s test missing now? I see only 3 e2e tests executed.
Also, I see that the test is not started at all: [Error](https://github.com/IndustryFusion/DigitalTwin/actions/runs/11544394192/workflow)
a step cannot have both the `uses` and `run` keys

yshashix commented:
> Why is the K8s test missing now? I see only 3 e2e tests executed. Also, I see that the test is not started at all: [Error](https://github.com/IndustryFusion/DigitalTwin/actions/runs/11544394192/workflow) a step cannot have both the `uses` and `run` keys

I have fixed the issue of [Error] where steps cant have both. and modified script.

wagmarcel commented on semantic-model/shacl2flink/pyenv_setup.sh:
I see a lot of tildes. Is there a reason why we install it in user's home instead of in DigitalTwin root directory?

yshashix replied:
When users install Miniconda, I thought to add it in the home directory so that they can use it not only specifically for Digital Twin but also to create Python environment across any other environment and vice versa. But when I re-think, it would be better to add it inside DigitalTwin repo as it is a requirement of DT.

wagmarcel replied:
So you will remove the pyenv_setup.sh or change it?

yshashix replied:
I have updated PR with the location of installation on the shacl2flink folder where pyenv_setup.sh runs.
